infrastructure/src/main/java/org/corfudb/infrastructure/management/ReconfigurationEventHandler.java

```diff
@@ -64,21 +74,53 @@ public boolean handleFailure(IReconfigurationHandlerPolicy failureHandlerPolicy,
 
     /**
      * Takes in the existing layout and a set of healed nodes.
-     * It first generates a new layout by adding the healed nodes to the existing layout.
-     * It then seals the epoch to prevent any client from accessing the stale layout.
-     * Finally we run paxos to update all servers with the new layout.
+     * The LayoutManagementView launches a workflow to heal the specified node. This call is
+     * blocked until the workflow is completed or aborted.
+     * The node is healed and added back to the layout with all components enabled i.e., with its
+     * layout, sequencer and log unit server enabled and as an active participant in the
+     * data replication view.
      *
      * @param currentLayout The current layout
-     * @param corfuRuntime  Connected corfu runtime instance
+     * @param runtime       Connected corfu runtime instance
      * @param healedServers Set of healed server addresses
      */
-    public boolean handleHealing(IReconfigurationHandlerPolicy failureHandlerPolicy,
-                                 Layout currentLayout,
-                                 CorfuRuntime corfuRuntime,
-                                 Set<String> healedServers) {
+    public boolean handleHealing(@Nonnull IReconfigurationHandlerPolicy failureHandlerPolicy,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "failureHandlerPolicy". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)